packages/beacon-node/src/monitoring/clientStats.ts

```diff
@@ -187,11 +187,46 @@ function createValidatorStats(): ClientStats {
     }),
     validatorActive: new MetricProperty({
       jsonKey: "validator_active",
-      metricName: "vc_indices_count",
+      metricName: "vc_active_validators_count",
       jsonType: JsonType.Number,
       defaultValue: 0,
       description: "Number of validator keys that are currently active",
     }),
+    validatorPending: new MetricProperty({
```

Comment:
this is the definition of client stats based on [this spec](https://docs.google.com/document/d/1qPWAVRjPCENlyAjUBwGkHMvz9qLdd_6u9DPZcNxDBpc/edit), only need to update `validatorActive` property